tools/awslint/lib/rules/resource.ts

```diff
@@ -152,7 +153,12 @@ resourceLinter.add({
   message: 'resources must represent all cloudformation attributes as attribute properties. missing property: ',
   eval: e => {
     for (const name of e.ctx.cfn.attributeNames) {
```

Comment:
So I naively tried to do this, but what do you do for all the situations where attributes and property names collide? For example, `CfnCapacityReservation` `availabiltyZone` is both attribute and property (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-capacityreservation.html#cfn-ec2-capacityreservation-availabilityzone).  No easy answer here with the naming convention from CloudFormation that I see? We can eliminate all `attrRef` and just use `ref`.

Reply:
What I mean here is that I still want generated L1 attribute properties to have the “attr” prefix but the array “attributeNames” in awslint should be the canonical attribute names without the prefix so it will be easy to work with in awslint

Reply:
alright -- I was I little dense there, I think I made the changes you wanted now.